src/components/InputPassword.js

```diff
@@ -91,7 +91,7 @@ class InputPassword extends Component {
   render() {
     const { requirementColour, requirements, ...props } = this.props;
     return (
-      <div>
+      <div {...props}>
```

Comment:
By that do you mean a plain `<div />` can't have props but a `<Container />` which is just `styled.div` can?